plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/DynamicFilteringJdbcConfig.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.trino.plugin.jdbc;
+
+import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigDescription;
+
+import javax.validation.constraints.NotNull;
+
+public class DynamicFilteringJdbcConfig
+{
+    private Long dynamicFilteringTimeout = 0L;
+
+    @NotNull
+    public Long getDynamicFilteringTimeout()
+    {
+        return dynamicFilteringTimeout;
+    }
+
+    @Config("dynamic_filtering_timeout")
```

Comment:
Config properties should be `-` instead of `_`. 
```suggestion
    @Config("dynamic-filtering.wait-timeout")
```